src/main/configurations/xxllnc/Common/xsl/CreateObjectLk01.xslt

```diff
@@ -0,0 +1,94 @@
+<xsl:stylesheet xmlns:xsl="http://www.w3.org/1999/XSL/Transform" xmlns:StUF="http://www.egem.nl/StUF/StUF0301" xmlns:xmime="http://www.w3.org/2005/05/xmlmime" version="2.0">
+    <xsl:output method="xml" version="1.0" encoding="UTF-8" indent="yes"/>
+    <xsl:param name="Base64Inhoud"/>
+    <xsl:param name="CaseReferenceNumber"/>
+    <xsl:param name="CaseDescription"/>
+    <xsl:param name="CaseTypeDescription"/>
+    <xsl:param name="CaseTypeCode"/>
+
+    <xsl:template match="/">
+        <object StUF:entiteittype="EDC">
```

Comment:
missing verwerkingssoort, which is mandatory in the schema. Can pass verwerkingsoort value by param.